web/templates/settings/filter_text_input.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="search_container">
+    <input type="text"  {{#if id}}id="{{id}}"{{/if}} class="search filter_text_input" placeholder="{{placeholder}}" aria-label="{{aria_label}}"/>
```

Comment:
Also, think there is an extra space after `type="text"` which can be removed.

Might be good to just pass ID for all the inputs or remove them from existing cases as well if they can be replaced by classes wherever used. Feels like an unnecessary "if" block that can be removed. Can be done as a follow-up as well.

Reply:
While making the changes for this PR some of those IDs seem to be unnecessary, so I will examine them carefully and clean them up if possible in my follow-up PR.
Thanks.